examples/src/test/scala/za/co/absa/fadb/examples/enceladus/DatasetSchemaSuite.scala

```diff
@@ -36,7 +36,7 @@ class DatasetSchemaSuite extends AnyWordSpec with Matchers {
   }
 
   "listSchemas" should {
-    "list the schemas" in {
+    "list the schemas" ignore {
```

Comment:
Perhaps when ignoring the existing tests, it would be nice to comment why -- what is the nature of the problem preventing (otherwise working) tests to be present normally.

Reply:
Good point. I explained in the commit message, but that is a pretty hidden info 😄